app/src/main/java/com/aptoide/uploader/MainPresenter.kt

```diff
@@ -23,11 +25,16 @@ class MainPresenter(val view: MainView, val accountManager: AptoideAccountManage
 
   override fun present() {
     compositeDisposable = CompositeDisposable()
-
+    handleLoginView()
     handleIntentEvents()
     onDestroyDisposeComposite()
   }
 
+  private fun handleLoginView() {
+    autoLoginManager.firstStoredUserCredentials
```

Comment:
This method is a Single and you are calling it without subscribing. I believe that what you are trying to achieve here is to pre-fetch the auto login user credentials but this should not be the way since right now it depends on how the method implemented and not in the method signature.

You should also not use the property syntax access for non-accessors (getter/setters) methods since it really hurts the readability (it doesn't look like it is calling a method, just accessing a property).

Reply:
Since its the first time the access to the Content provider is done, we need to fetch that info otherwise all the fields are null. 
Im looking into this now "This method is a Single and you are calling it without subscribing".

Regarding the property syntax, it was automatic in kotlin so I assumed it was the way calling methods worked on it and Im unsure what to do regarding this.

Reply:
I changed to subscribe the Single. But after thinking more about it, a can even change the method to be a void since i dont need it to return a Single.

Once again regarding the property syntax.. 
If the method is a Single and I dont use the property syntax, i get a warning telling me that im not using property syntax.
If the method is a void instead of a Single, then I dont get any warning using the normal syntax for the method